js/packages/mobile-wallet-adapter-protocol/src/jsonRpcMessage.ts

```diff
@@ -11,24 +12,29 @@ type JSONRPCResponse<TMessage> = {
 
 export async function encryptJsonRpcMessage(jsonRpcMessage: unknown, sharedSecret: SharedSecret) {
     const plaintext = JSON.stringify(jsonRpcMessage);
+    const sequenceNumber = new Uint8Array(SEQUENCE_NUMBER_BYTES);
     const initializationVector = new Uint8Array(INITIALIZATION_VECTOR_BYTES);
+    // TODO: populate sequence number (big-endian)
```

Comment:
@steveluscher could use your advice or assistance on these TODOs. I'm introducing the concept of message sequence numbers (see related MWA spec PR for details). With my weak JS skills, I'm not sure where the best place to track these values is, nor the best way to inject them into this function (parameters? globals?).